src/mlpack/methods/mean_shift/mean_shift.hpp

```diff
@@ -33,9 +33,11 @@ namespace meanshift /** Mean shift clustering. */ {
  * extern arma::mat data; // Dataset we want to run mean shift on.
  * arma::Row<size_t> assignments; // Cluster assignments.
  * arma::mat centroids; // Cluster centroids.
+ * bool forceConvergence; // Flag whether to force each centroid seed to
+ * converge regardless of maxIterations.
```

Comment:
Just for clarity of documentation I'd suggest setting `forceConvergence` to `true` or `false` in the example here (doesn't matter which).